python/pyarrow/tests/test_io.py

```diff
@@ -1073,6 +1073,8 @@ def _try_delete(path):
 
 
 def test_memory_map_writer(tmpdir):
+    if sys.platform == "emscripten":
+        pytest.xfail("Multiple memory maps to same file don't work on emscripten")
```

Comment:
Use a marker like in other places?

Reply:
And can maybe also be a skip instead of xfail (or are there prospects that this would someday work?)

Reply:
I was hoping that some day memory mapped files would actually work properly on emscripten. Right now they involve a copy, so multiple maps don't work right.

Reply:
And I'm not sure if it is worth a marker for a single weirdness

Reply:
Yes, fine to leave as is then